src/components/common/typography/index.ts

```diff
@@ -0,0 +1,27 @@
+import Heading1 from './Heading1';
+import Heading2 from './Heading2';
+import Heading3 from './Heading3';
+import Heading4 from './Heading4';
+import Body1 from './Body1';
+import Body2 from './Body2';
+import Body3 from './Body3';
+import Body4 from './Body4';
+import ButtonText1 from './ButtonText1';
+import ButtonText2 from './ButtonText2';
+import Caption1 from './Caption1';
+import Caption2 from './Caption2';
+
+export {
+  Heading1,
+  Heading2,
+  Heading3,
+  Heading4,
+  Body1,
```

Comment:
html 태그를 props로 받는다면

```suggestion
  Body1 : withTypographyBase('p', 'body4')
```

이런식으로 임포트해서 쓸 수 있을것같은데요?!